### PR TITLE
Lets use Text type for value column of TaskParameter table

### PR DESCRIPTION
**Pull request by oivoodoo**
## Description

Change `value` column type from `string(256)` to `text` because of the issues on insert to task parameters table.

[error](https://gist.github.com/oivoodoo/0fb78f5a041bb08939f30f4af10c00c6) 
## Motivation and Context

Found this issue from the log file in our production system.
## Have you tested this? If so, how?

Going to test on 17.10.2016 in the next our deploy.

### CODE REVIEW

**Comment by Tarrasch** [comment]
LGTM but if somebody else could review that would be good.

File: .gitignore [hunk 1]
```diff
@@ -170,3 +170,4 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+.python-version
```
Comment (kalvdans) [inline .gitignore]: This change in .gitignore is unrelated and should be in a separate patch.

**Comment by kwilcox** [comment]
Seems like you need to add an upgrade step to `_upgrade_schema` to migrate old schemas on a `luigi` upgrade?
